### DISCUSSION TITLE
Can step 2 in Setup guide be clarified?

### DISCUSSION THREAD

**Discussion by colinmiller-bard** [awslabs/aws-glue-libs#149] [Q&A]
`use copy-dependencies target in Apache Maven to download the jar from S3 to your local dev environment.`

An example command would be useful. Failing to get this running and I'm not sure I did this step right.

**Comment by AdrianoNicolucci** [comment]
I agree an example would be nice, this step is rather confusing, in the installation instruction page, this step is not even specified: https://docs.aws.amazon.com/glue/latest/dg/aws-glue-programming-etl-libraries.html. It's unclear if we need to do this step and if we do, what is the file from s3 would we pass to it?
